src/services/tiers.service.ts

```diff
@@ -148,14 +163,20 @@ export class TiersService {
     }
   }
 
-  async applyDriveFeatures(userWithEmail: User & { email: string }, tier: Tier): Promise<void> {
+  async applyDriveFeatures(
+    userWithEmail: User & { email: string },
+    customer: Stripe.Customer,
+    amountOfSeats: Stripe.InvoiceLineItem['quantity'],
+    tier: Tier,
+  ): Promise<void> {
     const features = tier.featuresPerService[Service.Drive];
 
     if (features.workspaces.enabled) {
+      if (!amountOfSeats) return;
```

Comment:
if features.workspaces.enabled = true, `amountOfSeats` would be set always, therefore this check is not needed.

Reply:
I have renamed the `amountOfSeats` parameter to `subscriptionSeats` for clarity. The idea is that this parameter represents the `amount` of subscription the user is paying for, which normally corresponds to the number of seats in the workspace plan - minimum of 3 seats per subscription. I have also added another check for seats less than the minimum.